release-team/role-handbooks/branch-manager/README.md

```diff
@@ -115,6 +116,42 @@ As the first build from the new release branch, publication this build does not
 Adding the --rc flag switches behavior on to building release candidates.  Again the gcbmgr/anago scripting automatically finds and increments the current build number.
 * `./gcbmgr stage release-1.12 --rc --build-at-head --nomock`
 
+In an perfect world, `rc.1` and the official release are the same commit. To get as close to that as we can the following thing should be considered:
+
+1. All the PRs in the milestone should merge to master
+
+   [`is:pr is:open milestone:v1.14`][pr-milestone-query] should be empty.
+
+   This means you (the release team) should really push on PRs towards the end
+   of code freeze or assess if they can be removed from the milestone and can
+   be punted to a next release.
+
+1. `rc.1` should be staged and released
+
+   Make sure that all the changes that have been merged in master also made it
+   into the release branch. You can also do a
+   [`branchff`](#branch-fast-forward) do see the state of the two branches and
+   potentially pull in last PRs into the release branch.
+
+   At this point in time master and the release branch are the same and master is
+   not merging anything new beacuase we are still in code freeze; therefore it
+   is safe to cut.
+
+1. Code Freeze can be lifted
+
+   Technically we can keep code freeze in place after `rc.1` was cut. However
+   we should see to lift code freeze relatively quickly after `rc.1`.
```

Comment:
_we should see to lift ..._ ? 

Not sure what is meant there.  Do you mean, check to see if we could lift the code freeze?

Reply:
Owh, after reading it several times, I think I got what you mean. _We should make effort to lift the code freeze ..._ ?